README.md

```diff
@@ -2,3 +2,30 @@
 This repository is used to document and define coding practices such as coding styles, writing commit messages and documentation.
 
 The code_conventions.md defines the general coding conventions, without going into language specifics. It is possible that certain languages have their own coding standards. If this is the case, an additional document is added, which provides addendums or changes to the general codestyle.
+
+
+## Structure
+
+Each language specific file describes overrides to the Ultiamker Generic style rules, which themselves are an override of some comprehensive base set of rules. This means the override stucture is as follows:
+
+```
```

Comment:
` ┌───────────────┐ 
    |                   UM Project                    |
    |____________________________________|`

Reply:
Pretend it's a rectangle ;)